src/components-examples/angular/tooltip/tooltip-hover/tooltip-hover-example.html

```diff
@@ -0,0 +1,20 @@
+<h4>Tooltip Component Usage</h4>
+<sbb-tooltip trigger="hover" [hoverShowDelay]="hoverShowDelay" [hoverHideDelay]="hoverHideDelay">
+  <span>This tooltip was shown on hover.</span>
+</sbb-tooltip>
+
+<h4>Tooltip Directive Usage</h4>
+<span
```

Comment:
better use `<p>` here because it would scale in 4k and 5k (span does not and stays very small).